packages/backend-common/src/reading/types.ts

```diff
@@ -83,6 +83,8 @@ export type ReadTreeResponseFactoryOptions = {
   etag: string;
   // Filter passed on from the ReadTreeOptions
   filter?: (path: string, info?: { size: number }) => boolean;
+  // First directory of a tar archive is stripped when set to true
```

Comment:
should probably make this a doc comment `/** */` and mention what the default value is

Reply:
I've moved the option to the `ReadTreeResponseFactory` as you advised.